test/onigumo_test.exs

```diff
@@ -34,7 +35,11 @@ defmodule OnigumoTest do
   defp get!(url) do
     %HTTPoison.Response{
       status_code: 200,
-      body: "Body from: #{url}\n"
+      body: body(url)
     }
   end
+
+  defp body(url) do
+    "Body from: #{url}"
```

Comment:
This was my mistake when merging. 😞 It should have already contain the newline.

```suggestion
    "Body from: #{url}\n"
```

Reply:
Fixed in https://github.com/Glutexo/onigumo/commit/35cce947a85f99114abad41b4314c08a61f42068.